docs/en/pact-functions.md

```diff
@@ -461,7 +461,7 @@ Return ID if called during current pact execution, failing if not.
 Obtain current pact build version.
 ```lisp
 pact> (pact-version)
-"4.10"
+"4.11"
```

Comment:
I do not think the docs have been regenerated in a while - there are some changes that are not related to what I have touched.

Reply:
In general: The readthedocs website is deprecated, *and* we need to _manually_ PR changes for docs.kadena.io, is this right @sstraatemans ?